squarelet/organizations/tests/test_models.py

```diff
@@ -660,6 +660,96 @@ def test_make_stripe_plan_group(self, organization_plan_factory, mocker):
             tiers_mode="graduated",
         )
 
+    @pytest.mark.django_db
+    def test_has_available_slots_non_sunlight_plan(self, plan_factory):
+        """Non-Sunlight plans always have available slots"""
+        plan = plan_factory(slug="professional", wix=False)
+        assert plan.has_available_slots() is True
+
+    @pytest.mark.django_db
+    def test_has_available_slots_sunlight_no_wix(self, plan_factory):
+        """Sunlight plans with wix=False have no limit"""
+        plan = plan_factory(slug="sunlight-basic", wix=False)
+        assert plan.has_available_slots() is True
+
+    @pytest.mark.django_db
+    def test_has_available_slots_sunlight_under_limit(
+        self, plan_factory, subscription_factory, settings
+    ):
+        """Sunlight wix plan under limit has available slots"""
+        settings.MAX_SUNLIGHT_SUBSCRIPTIONS = 15
```

Comment:
Settings should not be altered directly:
https://docs.djangoproject.com/en/5.2/topics/testing/tools/#overriding-settings

Reply:
It looks like modifying the settings fixture is a fine pattern with pytest-django: https://pytest-django.readthedocs.io/en/latest/configuring_django.html#overriding-individual-settings

However, I will update to the decorator for consistency.